runtime/src/tiered_storage.rs

```diff
@@ -205,8 +229,11 @@ mod tests {
         let tiered_storage_path = temp_dir.path().join("test_new_meta_file_only");
 
         {
-            let tiered_storage =
+            let mut tiered_storage =
                 TieredStorage::new_writable(&tiered_storage_path, HOT_FORMAT.clone());
+            // make the file reopenable later during the test.
+            tiered_storage.set_no_remove_on_drop();
+
```

Comment:
@brooksprumo: FYI.  This is an existing test where the test will fail without disabling remove-on-drop.